notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -74,9 +76,9 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you train a scikit-learn RandomForest model, save it in Vertex AI Model Registry and learn how to evaluate it through a Vertex AI pipeline job using `google_cloud_pipeline_components`.\n",
+        "In this tutorial, you train a scikit-learn RandomForest model, save it in Vertex AI Model Registry and learn how to evaluate it through a Vertex AI pipeline job using Google Cloud Pipeline Components Python SDK.\n",
```

Comment:
change "it" to "the model" (yes, repetitive, but should avoid using "it")
- In this tutorial, you train a scikit-learn RandomForest model, save the model in Vertex AI Model Registry and learn how to evaluate the model through a Vertex AI pipeline job using Google Cloud Pipeline Components Python SDK"